lib/iris/experimental/geobridge.py

```diff
@@ -0,0 +1,70 @@
+from geovista import Transform
+from geovista.common import VTK_CELL_IDS
+
+from iris.exceptions import CoordinateNotFoundError
+from iris.experimental.ugrid import Mesh
+
+
+def _get_coord(cube, axis):
+    try:
+        coord = cube.coord(axis=axis, dim_coords=True)
+    except CoordinateNotFoundError:
+        coord = cube.coord(axis=axis)
+    return coord
+
+
+def cube_faces_to_polydata(cube):
```

Comment:
Just discovered that I NEED `zlevel` to get [this plotting example](https://scitools-iris.readthedocs.io/en/stable/further_topics/ugrid/operations.html#plotting) to work; otherwise the face cuts through the sphere of the globe:

![image](https://github.com/SciTools/iris/assets/40734014/74a6b526-e3d2-429e-a234-de88885cc678)